### PR TITLE
1.19.4 update

### PR DESCRIPTION
**Pull request by Tides**
Another fairly big update~ This PR made some changes to the SoundEffect part of the api if there's something you don't like about let me know. Beside that this is as usual as it is, updated packet ids and added/modified some existing one to be inline with the protocol.

### CODE REVIEW

**Review by Jonpro03 — APPROVED** [review]
I actually read through/reviewed/tested everything. I couldn't find anything worth commenting. Excellent work. I'm going to propose a 10% salary increase for you on this PR alone

**Comment by Tides** [comment]
> I actually read through/reviewed/tested everything. I couldn't find anything worth commenting. Excellent work. I'm going to propose a 10% salary increase for you on this PR alone

😭

File: Obsidian.API/Crafting/CuttingRecipe.cs [hunk 1]
```diff
@@ -8,7 +8,7 @@ public sealed class CuttingRecipe : IRecipe
 
     public string? Group { get; init; }
 
-    public required Ingredient Result { get; init; }
+    public required Ingredient? Result { get; init; }
```
Comment (Craftplacer) [inline Obsidian.API/Crafting/CuttingRecipe.cs:11]: Why is this nullable?

**Review by Craftplacer — APPROVED** [review]

File: Obsidian.API/Crafting/CuttingRecipe.cs (hunk 1 above)
Comment (Tides) [reply Obsidian.API/Crafting/CuttingRecipe.cs:11]: I'll end up fixing this in another pr 👍🏼

**Pull request merged**